proto/storage/vulnerability.proto

```diff
@@ -38,7 +38,7 @@ message EmbeddedVulnerability {
   CVSSV2 cvss_v2 = 6;
   CVSSV3 cvss_v3 = 7;
   google.protobuf.Timestamp published_on = 9; // @gotags: search:"CVE Published On"
-  google.protobuf.Timestamp last_modified = 10;
+  google.protobuf.Timestamp last_modified = 10; // @gotags: hash:"ignore"
```

Comment:
I think we can hash ignore `suppressed`, `suppress_activation`, `suppress_expiry` and `state` as those are all deferral related fields and not provided by scanner.